src/applications/appeals/shared/components/IssueCard.jsx

```diff
@@ -205,7 +213,7 @@ export const IssueCard = ({
           <Header
             id={`issue-${index}-title`}
             className={titleClass}
-            data-dd-action-name="issue name"
+            data-dd-action-name="contestable issue name"
```

Comment:
Should `contestable` be capitalized like it's capitalized inside label up there?

Reply:
Making each action name unique so we know where the click is happening.